sync2jira/intermediary.py

```diff
@@ -103,7 +102,7 @@ def from_github(cls, upstream, issue, config):
             comments=comments,
             tags=issue['labels'],
             fixVersion=[issue['milestone']],
-            priority=None,
+            priority=issue['priority'],
```

Comment:
So, I guess this should be,
```suggestion
            priority=issue.get('priority'),
```
eh?  😊